plugins/kubernetes/app/models/watchers/cluster_pod_watcher.rb

```diff
@@ -2,16 +2,20 @@ module Watchers
   class ClusterPodWatcher < BaseClusterWatcher
     include Celluloid::Notifications
 
-    def initialize(client)
-      super(client.watch_pods)
+    def initialize(cluster)
+      super({
+        cluster: cluster,
+        watch_stream: cluster.client.watch_pods
+      })
     end
 
     private
 
     def handle_notice(notice)
+      debug notice.to_s
```

Comment:
Yep, noticed that in the meantime. I already removed this. Thanks for the heads up anyway.